src/test/java/org/opensearch/security/dlic/dlsfls/DlsPropsReplaceTest.java

```diff
@@ -58,5 +61,9 @@ public void testDlsProps() throws Exception {
         Assert.assertEquals(HttpStatus.SC_OK, (res = rh.executeGetRequest("/prop1,prop2/_search?pretty&size=100", encodeBasicHeader("prop_replace", "password"))).getStatusCode());
         System.out.println(res.getBody());
         Assert.assertTrue(res.getBody().contains("\"value\" : 3,\n      \"relation"));
+
+        Assert.assertEquals(HttpStatus.SC_OK, (res = rh.executeGetRequest("/prop-mapped/_search?pretty&size=100", encodeBasicHeader("prop_replace", "password"))).getStatusCode());
+        System.out.println(res.getBody());
+        Assert.assertTrue(res.getBody().contains("\"value\" : 1,\n      \"relation"));
```

Comment:
How does this ensure which doc is hit? Would it be more appropriate to assert `amount`?

Reply:
updated to assert the amount.